tfx/experimental/pipeline_testing/testdata/Evaluator/evaluation/eval_config.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
These test files are only for the chicago taxi pipelines.
Let's move all test files into tfx/experimental/pipeline_testing/examples/chicago_taxi_pipeline/testdata/....